### PR TITLE
Keep search field visible in long lists

### LINKED ISSUE #147: Keep search field visible
I'm trying it out with a lot of random extensions installed. Performance is good (M1 Pro though)

I think all it needs is something like:

```css
header {
  position: sticky;
  top: 0;
  background: inherit;
  padding: some padding probably
}
```

scroll

### PR DESCRIPTION
**Pull request by Anujjoshi3105**
Fixes #147 
The search field remains visible at the top of the viewport when scrolling.
image

### CODE REVIEW

**Review by fregante — CHANGES_REQUESTED** [review]
Thanks for the PR! However what I had asked is exactly what's needed, not `position: absolute`

```css
.header {
    position: sticky;
    top: 10px;
    background: gray;
    outline: solid 10px gray;
	z-index: 100
}
```

Just the values need to be adjusted, for example to use `var()` for the margin and the color.


scroll

File: source/style.css [hunk 1]
```diff
@@ -61,7 +61,10 @@ button:hover,
  */
 .header {
 	display: flex;
-	gap: 0.5em;
+	position: sticky;
+	z-index: 100;
+	top: var(--margin);
+	outline: solid var(--margin) #2b2c2c;
```
Comment (fregante) [inline source/style.css:67]: I think this fixes support for light mode too. Can you verify? Make send a new gif

```suggestion
	outline: solid var(--margin) light-dark(#fff, #292a2d);
```

**Review by fregante — APPROVED** [review]

**Comment by Anujjoshi3105** [comment]
Ligth and dark mode working 
light

**Comment by fregante** [comment]
Thank you!

**Pull request merged**